pisa/core/pipeline.py

```diff
@@ -84,9 +84,9 @@ class Pipeline(object):
 
     """
 
-    def __init__(self, config):
+    def __init__(self, config, skip_stages=None):
```

Comment:
The terminology in PISA is "stage" and "service". A "service" is the implementation of a "stage". E.g. `flux` is a stage; one "service" that provides flux would be `pi_mceq_barr`. For some stages, you could have more than one service, and it's possible to specify multiple stages (implemented by different services) in the same pipeline. I.e., `skip_services` is a more correct way to say this. And to specify a particular service, you have to provide `(stage, service)`.

Also, add documentation to the above "Parameters" section in the class docstring for this new argument.